src/QuickMarcEditor/QuickMarcEditorRows/QuickMarcEditorRows.js

```diff
@@ -374,7 +388,10 @@ const QuickMarcEditorRows = ({
                   }
                 </div>
 
-                <div className={styles.quickMarcEditorRowContent}>
+                <div
+                  className={styles.quickMarcEditorRowContent}
+                  ref={isLeader ? setLeaderInputWidth : null}
```

Comment:
If the `isLeader` variable is dedicated to setting the row content width only once to `leaderInputWidth`, rather than multiple times, then maybe it would be clearer to pass a function with the name `setRowContentWidthOnce` and rename `leaderInputWidth` to `rowContentWidth`.